src/app/map/_components/MapSection.tsx

```diff
@@ -1,11 +1,16 @@
 "use client";
-import { useState } from "react";
+import { Dispatch, SetStateAction, useState } from "react";
 import { MapScene } from "./MapScene";
 import { MapMenu } from "./MapMenu";
 
-export function MapSection() {
+type MapSectionProps = {
+  clickedRoom: string | null;
+  setClickedRoom: Dispatch<SetStateAction<string | null>>;
+};
+
+export function MapSection({ clickedRoom, setClickedRoom }: MapSectionProps) {
   const [floor, setFloor] = useState("Floor 1");
```

Comment:
I'm not sure if it's a good idea to store strings inside useState here, (for example if you'll get translations in the future you'll have to change that) I think it makes more sense to put a number `1` and if you want to display that, only then you'll format it to string

Reply:
it will be refactored in next PRs, when backend will add floors